### PR TITLE
Double all TC and TC prices

### PR DESCRIPTION
**Pull request by ghost**
The traitor uplink now has 20 telecrystals, uplink implants are 10 TC, nuke ops have 80. All prices in the uplink have been doubled. People who are NT Opposed will get 8 extra TC. I am thinking of changing a few of the prices but that's for another PR.

:cl:
 * rscadd: All telecrystals and uplink prices have been doubled.

### CODE REVIEW

**Comment by ghost** [comment]
I still think it should have been :100: rather than 20 but :ok_hand: this will allow for better flexibility with pricing shite

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -552,25 +552,25 @@ var/list/uplink_items = list()
...
 	item = /obj/item/weapon/storage/box/syndie_kit/imp_freedom
-	cost = 3
+	cost = 6
 
 /datum/uplink_item/implants/uplink
 	name = "Uplink Implant"
 	desc = "An implant injected into the body, and later activated using a bodily gesture to open an uplink with 5 telecrystals. The ability for an agent to open an uplink after their posessions have been stripped from them makes this implant excellent for escaping confinement."
 	item = /obj/item/weapon/storage/box/syndie_kit/imp_uplink
-	cost = 10
+	cost = 20
```
Comment (Shadowmech88) [inline code/datums/uplink_item.dm:561]: > uplink implants are 10TC

Comment (ghost) [reply code/datums/uplink_item.dm:561]: previously uplink implants cost 10 TC, but the actual uplink in the implant had 5 TC in it
now they cost 20 TC but the actual uplink has 10 TC in it

Comment (Shadowmech88) [reply code/datums/uplink_item.dm:561]: Oh, I see. My apologies.

**Comment by ComicIronic** [comment]
I thought we already did this at one point, or was that for wizard?

**Comment by Shadowmech88** [comment]
That was for wizard.

**Comment by ComicIronic** [comment]
Looks good.

**Comment by icantthinkofanameritenow** [comment]
I'm wondering why this was ever merged. Doubling both prices and crystals available effectively means the actual prices have not changed in the slightest. It simply makes no sense.

**Comment by AI-Is-Rogue** [comment]
@icantthinkofanameritenow The prices will be changed eventually, because there's not really much of a reason to just double everything and not change prices

**Comment by icantthinkofanameritenow** [comment]
@AI-is-rogue
Well, couldn't they have waited to merge this when said prices were ready to be changed? Because right now doubling everything is exactly what they did. 

I am seriously considering reverting it until those prices can be agreed on.

**Comment by ihadtoregisterforthis** [comment]
That's what #12550 is for.
Rescaling the points is a needed step for tweaking balance, but not a balance change.

**Comment by ComicIronic** [comment]
> revert a change for finer granularity because people haven't yet agreed what to do with it

wew
lad

Balance discussions take a while. We got a PR up on redoing uplink prices very soon after this was merged. The same thing happened for wiz.

**Comment by Intigracy** [comment]
This PR is basically inflation while increasing minimum wage to match it.

Prices don't really change how much they hurt your wallet because your income is the same.

You spent one dollar before, now you spend two, however before you earned 5 bucks an hour, now you earn 10.

But there might be items coming out for one dollar.

**Comment by Exxion** [comment]
> Getting upset over a change that does nothing except allow for greater flexibility in the future

**Comment by Intigracy** [comment]
Also there's precedent.

#11859

Doing the balance changes is gonna take a while but the idea itself is sound, so it's better they're done seperately.

**Pull request merged**